### PR TITLE
Create an empty profile modal for mentors

### LINKED ISSUE #541: Create a empty profile modal for mentors
**Is your feature request related to a problem? Please describe.**
- Need a mentor profile page to add to the scholarx 2020 page.

**Describe the solution you'd like**
- The profile card should contain only the image and the name. Remove the linkedin button from the current card.
- The card should be clickable. A modal should appear when clicking the card.
- It should be an empty modal for now.

**Describe alternatives you've considered**
- This page will popup when clicked on mentors name.

**Additional context**

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #541 

## Goals

- Create an empty profile modal for mentors

## Approach

- Added an empty modal.
- Remove the LinkedIn button from the current card. 

### Preview Link

https://pr-546-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/scholarx/2020/index.html [hunk 1]
```diff
@@ -554,18 +554,30 @@ <h5 class="mb-0">
...
+                                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                    <span aria-hidden="true">&times;</span>
+                                </button>
+                            </div>
+                            <div class="modal-body">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequatur dolor earum est eum ex harum incidunt non praesentium quisquam sequi.
+                            </div>
+                        </div>
+                    </div>
+                </div>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html]: Take this out from the moustache template. 
Place it at the end of the body.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
- the cursor should change to `pointer` when hovering

**Comment by anjula-sack** [comment]
Changes are applied.

File: src/main/webapp/scholarx/2020/index.html [hunk 2]
```diff
@@ -554,12 +571,8 @@ <h5 class="mb-0">
                 <div class="hover-profile-card">
                     <img src="images/{{image}}"
                          class="rounded-circle img-center img-fluid shadow shadow-lg--hover imgHover"
+                         data-toggle="modal" data-target="#profileModal" style="cursor: pointer"
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html]: write a custom CSS rule to change the cursor

ex: 

```
.hover-profile-card .img {
    cursor: pointer;
}
```

**Review by jayasanka-sack — DISMISSED** [review]

File: src/main/webapp/scholarx/2020/index.html (hunk 2 above)
Comment (jayasanka-sack) [reply src/main/webapp/scholarx/2020/index.html]: Usually, we do not use inline CSS

Comment (anjula-sack) [reply src/main/webapp/scholarx/2020/index.html]: on it.

**Review by Gravewalker666 — APPROVED** [review]

**Review by Gravewalker666 — APPROVED** [review]

**Pull request merged**